src/plugins/data/public/index_patterns/index_patterns/index_patterns.ts

```diff
@@ -21,7 +21,7 @@ import {
   SavedObjectsClientContract,
   SimpleSavedObject,
   IUiSettingsClient,
-  HttpServiceBase,
+  HttpSetup,
```

Comment:
`IndexPatterns` and `getSuggestionsProvider` services are being initialized in the `start` phase.
Could you use `HttpStart` here for it to make better sense?